Assets/MixedRealityToolkit.Extensions/Sharing/Localization/ISpatialCoordinate.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Numerics;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.Sharing
+{
+    /// <summary>
+    /// This represents a spatial coordinate that can then be used to convert position and rotation to and from this coordinate space.
+    /// </summary>
+    public interface ISpatialCoordinate
```

Comment:
>ISpatialCoordinate [](start = 21, length = 18)

General question based on this marker scenario: Do we want a ISpatialCoordinate to remain static in location? Is that a fundamental truth we want to build against? This marker location seems like a first instance where technically the spatial coordinate moves dynamically with device tracking. If we decide that all spatial coordinates should be considered static, we may need to dynamically create spatial coordinates for markers as devices move around a scene. If we decide that a spatial coordinate isn't static, we may want to add some functionality that allows coordinates to be static/non static. We may also want some functionality to be able to update the coordinate location by its creator?

Reply:
Yeah, I was thinking of having a IsTracked {get;set;} value for each ISpatialCoordinate as well. I feel because we are experimental, we can do this when the need arises.

---
In reply to: [270511687](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3732#discussion_r270511687) [](ancestors = 270511687)